components/panel-body/style.scss

```diff
@@ -0,0 +1,4 @@
+.components-panel-body {
+	padding: 15px;
+	box-shadow: 0 0 0 1px $light-gray-500;
```

Comment:
yep but not as easy, I'd like the `PanelBody`, `PanelHeader` and `Panel` to be used together (multiple headers, multiple bodies) and in the same time we could use them separately.  The three components would have this border and two of them could be nested inside the `Panel` component.

I think your proposal is possible but we'd probably have a lot of use-cases. I'll try it